neural_modelling/src/neuron/plasticity/stdp/synapse_dynamics_stdp_mad_impl.c

```diff
@@ -319,17 +318,16 @@ bool synapse_dynamics_initialise(
 }
 
 bool synapse_dynamics_process_plastic_synapses(
-        address_t plastic_region_address, address_t fixed_region_address,
+        address_t plastic_region_address,
```

Comment:
Could that use the correct type rather than casting in the function?

Reply:
Yes, by the magic of incomplete types.